scrapy_poet/injection.py

```diff
@@ -414,6 +441,7 @@ class MySpider(Spider):
     spider = MySpider()
     spider.settings = settings
     crawler.spider = spider
+    crawler.stats = load_object(crawler.settings["STATS_CLASS"])(crawler)
```

Comment:
Scrapy 2.11 changes broke this since `stats` was missing.

Reply:
Ouch, I've already fixed it in master before looking here.

Reply:
But it's good to see we did it in (kinda) the same way!